src/viam/sdk/module/service.cpp

```diff
@@ -115,8 +113,8 @@ struct ModuleService::ServiceImpl : viam::module::v1::ModuleService::Service {
             Registry::get().lookup_model(cfg.name());
         if (reg) {
             try {
-                const std::shared_ptr<Resource> res = reg->construct_resource(deps, cfg);
-                manager->replace_one(cfg.resource_name(), res);
+                const std::shared_ptr<Resource> resource = reg->construct_resource(deps, cfg);
```

Comment:
Windows throws a shadowing warning here which gets promoted to an error because we have `res` above.